common/client-core/src/client/statistics_control.rs

```diff
@@ -0,0 +1,174 @@
+// Copyright 2024 - Nym Technologies SA <contact@nymtech.net>
+// SPDX-License-Identifier: GPL-3.0-only
```

Comment:
isn't client code meant to be licensed under Apache?

Reply:
I honestly don't know how we're licensing what, so you tell me

Reply:
Fixed in https://github.com/nymtech/nym/pull/5107/commits/2b6498da031f014035b64641294332ed5bac5eb3, along with other places I could think of